keyboards/7splus/keymaps/salicylic/keymap.c

```diff
@@ -0,0 +1,130 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file should have a license header, as the default keymap does.

Reply:
Thank you. I forgot about it!